### ISSUE TITLE
Poll frequency question

### ISSUE THREAD

**Issue by jpoeppelman1** [hepcat72/node-red-contrib-life360#4]
Just a few questions..  Does this node poll for location status or is the application event driven meaning websockets?  If poll, what is the poll frequency?

**Comment by hepcat72** [comment]
It polls at an interval of 15 seconds:

https://github.com/hepcat72/node-red-contrib-life360/blob/3a8654143b39f047224905d74637151de4f3cb53/nodes/server.js#L42-L47

**Comment by jpoeppelman1** [comment]
> It polls at an interval of 15 seconds:
> 
> https://github.com/hepcat72/node-red-contrib-life360/blob/3a8654143b39f047224905d74637151de4f3cb53/nodes/server.js#L42-L47

Anyway we can make this configurable through Node Red configuration node?

**Comment by hepcat72** [comment]
I agree having that as a configurable option would be nice. I don't however have the time to do it, TBH.  (You wouldn't believe how swamped I am on multiple fronts.)  However, I will take a look at any new feature PR if you would like to take a stab at it.

**Comment by jpoeppelman1** [comment]
Hello!  Just wondering if you now have the time to make this a configurable option within the config node?  :)

**Comment by hepcat72** [comment]
I don't even have time to set up my new pi 4 which has been sitting for about a year unopened. And I've been taking 2 vacation days a week for over a month to try and wrap up a DIY bathroom remodel. The wife keeps bugging me about that.

I'd look at a PR if you submit one though.

**Comment by jpoeppelman1** [comment]
I'm not a node.js expert so I probably won't be able to fix this.  I can venmo you some money if you fix this if that would help?  :)

**Comment by hepcat72** [comment]
Lol. Thanks for the offer. Though I don't think there's an amount of money corresponding to what this feature is worth that would tempt me to divert attention from my other projects. Though if you come help me with my bathroom remodel, that may be worth it for me. ;)

What's your native programming language? I dove into Node.js when I started this project. My native language is perl. Though I've been doing more and more JavaScript since I did the Life360 node.

**Comment by brianmay** [comment]
Is disappointing that we have to resort to polling and there isn't some sort off streaming API. But I guess that is all that life360 provides. Or maybe all that has been reversed engineered so far.

**Comment by jpoeppelman1** [comment]
Hi.  On a similar note, what is the bearer token expiration against the Life360 API?  Just curious how often this needs to refreshed or if it ever expires?

**Comment by hepcat72** [comment]
Oh yeah! I remember encountering something like that but I didn't save it. Probably should have set it up in my reminders.

**Comment by jpoeppelman1** [comment]
Does the bearer token ever expire?
